datatransfer/impl/graphsync/graphsync_impl_test.go

```diff
@@ -407,176 +407,6 @@ func (sv *stubbedValidator) verifyExpectations(t *testing.T) {
 	}
 }
 
-func TestSendResponseToIncomingRequest(t *testing.T) {
```

Comment:
Nb: I'm not sure I totally agree with splitting this into the receiver file, cause you're not directly testing the receiver. that's ok though. I guess that's the code that's mostly getting exercised.

Reply:
agree, but symmetry I guess?